dataclients/kubernetes/kube.go

```diff
@@ -845,8 +857,21 @@ func (c *Client) ingressToRoutes(items []*ingressItem) ([]*eskip.Route, error) {
 			"ingress": fmt.Sprintf("%s/%s", i.Metadata.Namespace, i.Metadata.Name),
 		})
 
+		redirect.initCurrent(i.Metadata)
+
 		if r, ok, err := c.convertDefaultBackend(i); ok {
 			routes = append(routes, r...)
+
+			// we cannot control the redirect for the default backends, because
```

Comment:
Can you link the issue or comment in an issue you have for this commented code, too?

Reply:
https://github.com/zalando/skipper/issues/801